geopandas/io/sql.py

```diff
@@ -63,3 +65,111 @@ def load_geom(x):
                 crs = {"init": "epsg:{}".format(srid)}
 
     return GeoDataFrame(df, crs=crs, geometry=geom_col)
+
+
+def write_postgis(df, name, con, **kwargs):
+    """
+    Write a GeoDataFrame to a PostGIS table.
+
+    Tables can be newly created, appended to, or overwritten.
+    The type and the crs of the GeoDataFrame geometry is converted automatically to match PostGIS format.
+    This behavior can be changed as shown in the Examples section.
+
+    Parameters
+    ----------
+    df : GeoDataFrame
+    name : string
+        Name of PostGIS table
+    con : sqlalchemy.engine.Engine or DB connection object
+        Active connection to the database.
+    kwargs :
+        passed to pandas.to_sq. See documentation for available parameters
+
+    Raises
+    ------
+    ValueError
+        When the table already exists and `if_exists` is 'fail' (the
+        default).
+
+    See Also
+    --------
+    pandas.read_sql : pass kwargs to this function
+
+    References
+    ----------
+    .. [1] http://docs.sqlalchemy.org
+    .. [2] https://www.python.org/dev/peps/pep-0249/
+    .. [3] https://geoalchemy-2.readthedocs.io/en/latest/
+
+    Notes
+    ------
```

Comment:
For nice auto-doc building, please align the number of `-` with the number of characters in the section header.